viewer-prototype/src/browser/trace-explorer/trace-explorer-widget.tsx

```diff
@@ -112,66 +116,72 @@ export class TraceExplorerWidget extends ReactWidget {
-                    <List
-                        height={300}
-                        width={300}
-                        rowCount={outputsRowCount}
-                        rowHeight={50}
-                        rowRenderer={this.outputsRowRenderer} />
+            return <div className='trace-explorer-container'>
+                <ReactModal isOpen={this.showShareDialog} onRequestClose={this.handleShareModalClose}
+                    ariaHideApp={false} className='sharing-modal' overlayClassName='sharing-overlay'>
+                    {this.renderSharingModal()}
+                </ReactModal>
+                <div className='trace-explorer-opened'>
+                    <div className='trace-explorer-panel-title' onClick={this.updateOpenedExperiments}>
+                        {this.OPENED_TRACE_TITLE}
+                    </div>
+                    <div className='trace-explorer-panel-content'>
+                        <List
+                            height={300}
+                            width={300}
+                            rowCount={this.openedExperiments.length}
+                            rowHeight={50}
+                            rowRenderer={this.experimentRowRenderer} />
+                    </div>
                 </div>
-            </div>
-            <div className='trace-explorer-tooltip'>
-                <div className='trace-explorer-panel-title'>
-                    {'Time Graph Tooltip'}
+                <div className='trace-explorer-analysis'>
+                    <div className='trace-explorer-panel-title'>
+                        {this.ANALYSIS_TITLE}
+                    </div>
+                    <div className='trace-explorer-panel-content'>
+                        <List
+                            height={300}
+                            width={300}
+                            rowCount={outputsRowCount}
+                            rowHeight={50}
+                            rowRenderer={this.outputsRowRenderer} />
+                    </div>
                 </div>
-                <div className='trace-explorer-panel-content'>
-                    {this.renderTooltip()}
+                <div className='trace-explorer-tooltip'>
+                    <div className='trace-explorer-panel-title'>
+                        {'Time Graph Tooltip'}
+                    </div>
+                    <div className='trace-explorer-panel-content'>
+                        {this.renderTooltip()}
+                    </div>
                 </div>
+            </div>;
+        }
+
+        return <div className='theia-navigator-container' tabIndex={0}>
+            <div className='center'>{'You have not yet opened a trace.'}</div>
+            <div className='open-workspace-button-container'>
+                <button className='theia-button open-workspace-button' title='Select a trace to open'
+                    onClick={this.handleOpenTrace}>{'Open Trace'}</button>
```

Comment:
Could you post an updated screenshot of this?

Reply:
Sure.